nip-weather.md

```diff
@@ -0,0 +1,114 @@
+
+```jsonc
+{
+  "kind": 10xxx,
+  "tags": [
+    ["name", "Backyard Station"],
+    ["g", "9q5h"],
+    ["elevation", "52"],
+    ["power", "mains"],
+    ["connectivity", "wifi"],
+    ["sensor", "temp", "DHT11"],
+    ["sensor", "humidity", "DHT11"],
+    ["sensor", "pm25", "PMS5003"]
+  ],
+  "content": ""
+}
+```
+
+Tags:
+- `name` (optional): Human-readable station name
+- `g` (optional): Geohash for location indexing (see [NIP-52](52.md))
+- `elevation` (optional): Elevation in meters above sea level
+- `power` (optional): Power source type. Common values: `mains`, `solar`, `battery`, `solar_battery`, `usb`
+- `connectivity` (optional): Connectivity type. Common values: `wifi`, `cellular`, `ethernet`, `lora`, `satellite`
+- `sensor` (repeatable): Sensor types available with model identifier. Format: `["sensor", "<type>", "<model>"]`. Used for filtering/discovery (relays can index these).
+- `t` (optional): Hashtag for discovery. SHOULD include `["t", "weather"]` for relay indexing.
+- Other tags as needed
+
+To group multiple stations together, use [NIP-51](51.md) lists. `kind:30001` (generic lists) can be used with `p` tags referencing station pubkeys. Example:
+
+```jsonc
+{
+  "kind": 30001,
+  "tags": [
+    ["d", "my-stations"],
+    ["title", "My Weather Stations"],
+    ["p", "<station1-pubkey>"],
+    ["p", "<station2-pubkey>"]
+  ]
+}
+```
+
+## Weather Station Readings (`kind:xxxx`)
+
+Regular events containing sensor readings. In the regular range (1000-9999), all readings are stored by relays, enabling historical queries (e.g., last hour, last day). All sensor data is stored in tags.
+
+Sensor readings use 3-parameter tags: `[sensor_type, value, model]`. The third parameter identifies the sensor model, enabling cross-station comparison and supporting multi-sensor setups where the same sensor type may use different models.
+
+```jsonc
+{
+  "kind": xxxx,
+  "tags": [
+    ["t", "weather"],
+    ["a", "10xxx:<pubkey>:", "<relay-url>"],
+    ["temp", "22.5", "DHT11"],
+    ["humidity", "65.2", "DHT11"],
+    ["pm1", "8", "PMS5003"],
+    ["pm25", "12", "PMS5003"],
+    ["pm10", "15", "PMS5003"],
+    ["air_quality", "627", "MQ-135"]
+  ],
+  "content": ""
+}
+```
+
+Tags:
+- `t` (optional): Hashtag for discovery. SHOULD include `["t", "weather"]` for relay indexing.
+- `a` (optional): Reference to the station metadata event (`10xxx:<pubkey>:` - note trailing colon for replaceable events). This links the reading to its station.
+- Sensor reading tags (repeatable, 3-parameter format `[sensor_type, value, model]`):
+  - `temp`: Temperature in Celsius (°C)
+  - `humidity`: Relative humidity (0-100%)
+  - `pm1`, `pm25`, `pm10`: Particulate matter concentration in µg/m³
+  - `air_quality`: Raw analog sensor value (0-1023, uncalibrated)
+  - Other sensor tags as needed
+
+The third parameter (model) identifies the sensor model (e.g., "DHT11", "PMS5003", "MQ-135"), enabling:
```

Comment:
It would be good to define if this string follows a specific format or is just generally human readable.
If its designed to be aggregated across stations then it would be good to have it standardized so there is as much interoperability as possible